debian/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
+MAINTAINER Shaun Jackman <sjackman@gmail.com>
```

Comment:
1. This syntax is deprecated. `LABEL maintainer="First Last email@addre.\ss"`
2. Put your name here :)